### DISCUSSION TITLE
BL touch issue

### DISCUSSION THREAD

**Discussion by yuta4** [mriscoc/Ender3V2S1#1214] [Q&A]
Hi, I have an issue with build mesh functionality. After changing the nozzle and readjusting z offset it started produce weird result - all points have negative offset, including central point:
![IMG_20240113_121333](https://github.com/mriscoc/Ender3V2S1/assets/9968566/991a42e7-749a-47b6-b437-bbef81a9f058)
![IMG_20240113_121344](https://github.com/mriscoc/Ender3V2S1/assets/9968566/40c6cc1d-2e65-4263-85a8-3b562d002db4)
If my knowledge is correct - at least one point (I guess central, home point) should be treated as 0 and all other will have some offset comparing to it, and we should never have the result like I do.
After some random time of restarts I finally get correct result (without other physical or software adjustments from my side, just pressing "auto build mesh" multiple times, including after restart printer, which usually doesn't help)
![IMG_20240113_122608](https://github.com/mriscoc/Ender3V2S1/assets/9968566/7e11ca30-e07f-4a61-b02c-bf5e953f4dbd)
M48 probe gives fine results for my BL touch, so it is unlikely that issue is related to it

**Comment by mriscoc** [comment]
It seems to be a mechanical problem, the point of the mesh more close to the center should be close to 0 (green), use a TJC printer firmware  to be able to see the numerical values of the mesh: https://github.com/mriscoc/Special_Configurations/releases/tag/tjc

**Reply by yuta4** [reply]
@mriscoc by mechanical problem you mean problem with CR touch?
Thanks for firmware suggestion, now I have numbers too :)
